src/content/docs/cheerpj3/10-getting-started/03-JNLP.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Run a JNLP
+description: Run a JNLP application in the browser (setup)
+---
+
+This quickstart tutorial will take you step by step on the things you need to setup before running your JNLP app in the browser with CheerpJ.
+
+You will need:
+
+- The application `.jnlp` file.
+
+## 1. The `.jnlp` file
+
+The `.jnlp` file contains the specifics of how to launch your Java application. The usual pipeline starts when this file is triggered from a website so it is passed to the user's local JNLP client which downloads the application `.jar` files and further resources. Finally, the app is executed with the local JRE installation. **With CheerpJ it is possible to run this application in the browser sandbox, no Java installation required.**
+
+Your `.jnlp` file might look something like the example below. There are two essential elements you need to find:
+
+1. The `.jar` files specified under the `<resources>` element, usually indicated with a `<jar>` or `<nativelib>` tags.
+2. Your application type. Seek for ` <application-desc>` or `<applet-desc>` tag.
+3. You may need the `codebase` URL at `<jnlp>`
```

Comment:
It's awkward having a third point here given that you just said they need "two essential elements"